langchain4j-open-ai/src/main/java/dev/langchain4j/model/openai/OpenAiResponsesStreamingChatModel.java

```
@@ -0,0 +1,996 @@
+                    .addHeader("Content-Type", "application/json")
+                    .addHeader("Accept", "text/event-stream");
+
+            if (organizationId != null) {
+                requestBuilder.addHeader(OPENAI_ORGANIZATION_HEADER, organizationId);
+            }
+
+            String requestBody = OBJECT_MAPPER.writeValueAsString(payload);
+            logger.debug("Sending OpenAI Responses payload: {}", requestBody);
+
+            var request = requestBuilder.body(requestBody).build();
+
+            var streamingHandle = new ResponsesStreamingHandle();
+            httpClient.execute(
+                    request,
+                    new DefaultServerSentEventParser(),
+                    new ResponsesApiEventListener(handler, streamingHandle));
+
+        } catch (Exception e) {
+            logger.error("Exception in doChat", e);
+            safeOnError(handler, e);
+        }
+    }
+
+    private static void safeOnError(StreamingChatResponseHandler handler, Throwable error) {
+        try {
+            handler.onError(error);
+        } catch (Exception e) {
+            logger.warn("Exception thrown by onError handler, ignoring", e);
+        }
+    }
+
+    private List<Map<String, Object>> toResponsesMessages(ChatMessage msg) {
+        if (msg instanceof SystemMessage systemMessage) {
+            return List.of(createMessageEntry(ROLE_SYSTEM, List.of(createInputTextContent(systemMessage.text()))));
+        } else if (msg instanceof UserMessage userMessage) {
+            List<Map<String, Object>> contentEntries = new ArrayList<>();
+            for (Content content : userMessage.contents()) {
+                if (content instanceof TextContent textContent) {
+                    contentEntries.add(createInputTextContent(textContent.text()));
+                } else if (content instanceof ImageContent imageContent) {
+                    contentEntries.add(createInputImageContent(imageContent.image()));
+                }
```

please throw an exception if content type is not supported, instead of silently ignoring it

---

done